docs/DeveloperGuide.adoc

```diff
@@ -297,69 +312,430 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
-|=======================================================================
+|`* * *` |student |view information related to a module  |prepare for each module
+
+|`* * *` |student |view tasks |complete them
+
+|`* * *` |student |view facilitators' information |contact them when I need help
+
+|`* * *` |student |edit a module’s description |modify the module’s description
+
+|`* * *` |student |edit classes |keep my classes up to date
+
+|`* * *` |student |edit a task |keep my tasks up to date
+
+|`* * *` |student |edit a facilitator’s information |keep their contact details up to date
+
+|`* * *` |student |delete a module |use the App for different semesters
+
+|`* * *` |student |delete a class |remove classes that I am no longer in
+
+|`* * *` |student |delete a task |remove tasks that I no longer need to track
+
+|`* * *` |student |delete a facilitator’s information |remove information that I no longer need
+
+|`* * *` |busy student |view schedule for the current week |prepare for them
+
+|`* * *` |busy student |view schedule for the upcoming week |prepare for them
+
+|`* * *` |new user |view all commands |learn how to use them
+
+|`* * *` |new user |view commands for a specific feature |learn how to use them
+
+|`* * *` |user |get reminded about how commands work |recall the commands
+
+|`* * *` |user |import and export data |easily migrate the data to another computer
+
+|`* *` |student |find a facilitator by name |locate details of facilitators without having to go through the entire list
+
+|`* *` |student |find tasks by date |keep track of tasks on a particular date
+
+|`* *` |student |find upcoming tasks |prioritise them
+
+|`* *` |busy student |find empty slots in my schedule |manage my time easily
 
-_{More to be added}_
+|`*` |student |mark a task as done |not take note of them anymore
+
+|`*` |student |add a priority level to a task |prioritise my tasks
+
+|`*` |student |tag my tasks |categorise them
+
+|`*` |student |see countdown timers |be reminded of deadlines
+
+|`*` |busy student |receive reminders about deadlines and events the next day |take note of them
+
+|`*` |student |mass delete the modules |delete them quickly once the semester is over
+
+|`*` |advanced user |use shorter versions of a command |type a command faster
+
+|`*` |careless user |undo my commands |undo the mistakes in my command
+
+|`*` |visual user |see a clear <<gui, GUI>> |navigate the App more easily
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `Mod Manager` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== Use case: UC01 - Add module
+
+*<<mss, MSS>>*
+
+1.  User requests to add a module.
+2.  User provides the module code and description of the module.
+3.  Mod Manager adds the module.
```

Comment:
Does Mod Manager save data into the system? If so, can add in a step for system saving data.